code/DebugBar.php

```diff
@@ -234,8 +235,8 @@ public static function includeRequirements()
         $renderer = $debugbar->getJavascriptRenderer();
 
         // We don't need the true path since we are going to use Requirements API that appends the BASE_PATH
-        $renderer->setBasePath(DEBUGBAR_DIR . '/assets');
-        $renderer->setBaseUrl(DEBUGBAR_DIR . '/assets');
+        $renderer->setBasePath(Director::makeRelative(ModuleLoader::getModule('lekoala/silverstripe-debugbar')->getResource('assets')->getURL()));
```

Comment:
As above, use getPath() / getRelativePath() for a path, don't try to infer from getURL().